internal/renderers/skia/itemrenderer.rs

```diff
@@ -117,6 +128,7 @@ impl<'a> SkiaItemRenderer<'a> {
             }
             _ => None,
         }
+        .and_then(|shader| Some(func(paint, Some(shader))))
```

Comment:
If i'm not mistaken, this could be written 

```suggestion
        .map(|shader| func(paint, Some(shader)))
```